zellij-tile/README.md

```diff
@@ -0,0 +1,5 @@
+# zellij-tile
+
+Source code of the `ZellijPlugin` trait, a core detail for implementing your own plugin. With this crate you can interact with the zellij session through [shims](https://docs.rs/zellij-tile/latest/zellij_tile/shim/index.html)
```

Comment:
I think it is better to call it what it calls itself in the docs : "it is the Rust API for developing plugins". So I suppose "zellij-tile-utils" is linked to that.